examples/dev/Alpha_Config_UC

```diff
@@ -0,0 +1,39 @@
+## Use Case (UC): Running default config on the JUAMI Potentiostat for Alpha release.
+Intended User: Graduate student (GS) familiar with dev team and basic electrochemistry concepts. The instructions in this UC assume that the student has a resistor they can attach clips to.
+Assumes little to no knowledge of python. Assumes access to a Windows computer.
+
+### Before Installation: 
+1. Member of dev team gives GS a JUAMI Potentiostat with a 1kOhm resistor.
+1. GS installs python on their computer.
```

Comment:
I think you should change python to miniconda to be more specific.

Reply:
I agree.   python and miniconda are different.  Let's make instructions that can be followed step by step.  There should probably be a pre-step that asks the user to check their system to see if miniconda or Anaconda is installed.  They could just open a terminal and type "conda" for example.

We have to decide whether we want to make them use the miniconda command prompt.  It changes how they do the installation.  There are arguments for doing it either way so I am not sure....we could discuss more on a call.